types/next-pwa/package.json

```diff
@@ -1,6 +1,9 @@
 {
   "private": true,
   "dependencies": {
+    "@types/node": "*",
+    "@types/react": "*",
+    "@types/react-dom": "*",
```

Comment:
If you copied these from my DT monorepo branch, I don't think they are needed here; in `master` we're still using `typesRoot` so these won't have any effect.

Reply:
Yeah, there seems to be an issue with `typesRoot` resolution. I'm looking into it.